spynnaker/pyNN/models/neuron/synapse_dynamics/synapse_dynamics_stdp.py

```diff
@@ -207,20 +213,30 @@ def get_plastic_synaptic_data(
             fixed_plastic_rows = self._pad_row(fixed_plastic_rows, 2)
         fp_data = self.get_words(fixed_plastic_rows)
 
-        # Get the plastic data
+        # Get the plastic data by inserting the weight into the half-word
+        # specified by the synapse structure
         synapse_structure = self._timing_dependence.synaptic_structure
-        plastic_plastic = synapse_structure.get_synaptic_data(connections)
-        plastic_headers = numpy.zeros(
-            (n_rows, self._n_header_bytes), dtype="uint8")
+        n_half_words = synapse_structure.get_n_half_words_per_connection()
+        half_word = synapse_structure.get_weight_half_word()
+        plastic_plastic = numpy.zeros(
```

Comment:
I hadn't noticed this before now, but "plastic_plastic" seems a somewhat unnecessary name - what's wrong with simply using "plastic" here instead?